src/network/networkd-network.c

```diff
@@ -303,7 +318,7 @@ int network_load_one(Manager *manager, OrderedHashmap **networks, const char *fi
                 .manager = manager,
                 .n_ref = 1,
 
-                .required_for_online = true,
+                .required_for_online = -1,
```

Comment:
Could you update the branch `if (isempty(rvalue))` in `config_parse_required_for_online()` to match this default?

Reply:
ack, updated in new push